# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/test/mochitest.ini
# Commit: b72d1d6131cc
# Full Hash: b72d1d6131cce6845af6ffd6f83de7f993930961
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-26 09:42:55
# Description:
#   Bug 1576154 - Check mSVGDocumentWrapper when trying to report use counters; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43256
# ==============================================================================

diff -r 929421783c5b -r b72d1d6131cc dom/base/test/mochitest.ini
--- a/dom/base/test/mochitest.ini	Sun Aug 25 23:22:11 2019 +0000
+++ b/dom/base/test/mochitest.ini	Sun Aug 25 22:45:33 2019 +0000
@@ -52,6 +52,7 @@
   bug704320.sjs
   bug704320_counter.sjs
   bug819051.sjs
+  bug1576154.sjs
   chrome/bug418986-1.js
   copypaste.js
   delayedServerEvents.sjs
@@ -244,6 +245,7 @@
   !/dom/animation/test/testcommon.js
   !/dom/events/test/event_leak_utils.js
   ../../../browser/extensions/pdfjs/test/file_pdfjs_test.pdf
+  green.png
 
 [test_anchor_area_referrer.html]
 [test_anchor_area_referrer_changing.html]
@@ -634,6 +636,7 @@
 [test_bug1472427.html]
 [test_bug1499169.html]
 skip-if = toolkit == 'android' # Timeouts on android due to page closing issues with embedded pdf
+[test_bug1576154.html]
 [test_caretPositionFromPoint.html]
 [test_change_policy.html]
 [test_clearTimeoutIntervalNoArg.html]